modules/os2forms_forloeb/src/MaestroHelper.php

```diff
@@ -389,7 +389,10 @@ private function sendNotificationDigitalPost(
       $senderLabel = $subject;
       $messageLabel = $subject;
 
-      $recipientLookupResult = $this->digitalPostHelper->lookupRecipient($recipient);
+      // Remove all non-digits from recipient identifier.
+      $recipientIdentifier = preg_replace('/[^\d]+/', '', $recipient);
+
+      $recipientLookupResult = $this->digitalPostHelper->lookupRecipient($recipientIdentifier);
```

Comment:
I have a suggestion which is related directly with the changed lines:

- it makes sense to explain in the doc what parameter is expected/allowed [MaestroHelper.php#L359](https://github.com/OS2Forms/os2forms/blob/6e83da36ccef46813abb774dcbd2d19e15cc252c/modules/os2forms_forloeb/src/MaestroHelper.php#L359)